src/auth/component.ts

```diff
@@ -70,6 +70,18 @@ export default class ngeoAuthComponent extends LitElementI18n {
         },
       })
     );
+    this.subscriptions_.push(
+      configuration.getConfig().subscribe({
+        next: (configuration: Configuration) => {
+          if (configuration) {
```

Comment:
Is this `if` really useful ? Maybe at the initialization ? In this case, you can use a `Subject` instead of a behavior Subject. It's the same without the default value (`undefined` here).

And then, we could add a check in the `configuration.setConfig()` function to avoid passing a null ro undefined value. (or more check if necessary).

---

You can test the differences in this example I've made:https://rxjs-playground.github.io/#/?js=const%20subject%20%3D%20new%20Rx.Subject%28%29%3B%0A%0Asubject.subscribe%28%7Bnext%3A%20%28val%29%20%3D%3E%20console.log%28%60subject%20says%20%24%7Bval%7D%60%29%7D%29%3B%0Asubject.next%281%29%3B%0Asubject.next%282%29%3B%0A%0Aconst%20behaviorsubject%20%3D%20new%20Rx.BehaviorSubject%28undefined%29%3B%0A%0Abehaviorsubject.subscribe%28%7Bnext%3A%20%28val%29%20%3D%3E%20console.log%28%60behaviorsubject%20says%20%24%7Bval%7D%60%29%7D%29%3B%0Abehaviorsubject.next%28%27a%27%29%3B%0Abehaviorsubject.next%28%27b%27%29%3B: